From c7cdb189bdcbf2e1c321558f52685be868121716 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 3 Jul 2019 18:53:25 +0200
Subject: [PATCH] nitrogen8mm.h: Enable env import and saveenv

These configs are enabled in the configs file
but get disabled in the board configuration file

Enable them to be able to load resinOS_uEnv.txt

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/nitrogen8mm.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/nitrogen8mm.h b/include/configs/nitrogen8mm.h
index fcb047649b..f190c9b970 100644
--- a/include/configs/nitrogen8mm.h
+++ b/include/configs/nitrogen8mm.h
@@ -54,8 +54,8 @@
 /* Flat Device Tree Definitions */
 #define CONFIG_OF_BOARD_SETUP
 
-#undef CONFIG_CMD_EXPORTENV
-#undef CONFIG_CMD_IMPORTENV
+
+
 #undef CONFIG_CMD_IMLS
 
 #undef CONFIG_CMD_CRC32
-- 
2.17.1

